shelley/pool-ranking/pool-ranking.tex

```diff
@@ -28,78 +29,119 @@
+  account the rewards that a newly formed stake pool would yield when it reaches saturation (\cite{bkks2018} describes this in detail).
+  The specific relationship to the Cardano design is explained in the IOHK delegation design document \cite[Section 5.6]{delegation_design}.
+  \item
+    A stake pool will sometimes have the opportunity to produce blocks,
+    but fails to do so due to issues such as a poor network connection or untimely downtime. The rate at
+    which a pool uses their opportunities to make blocks, which we will call its \emph{hit rate}, affects rewards.
+    It needs to be taken into account.
+  \item
+    The na\"{i}ve approach only considers a single set of rewards.  Delegating on this basis would create significant
+    instability in the system, with delegators pursuing short-term rewards over the long-term health of the network.
+    In order to avoid this issue and so create more stability, it is necessary to consider a pool's historic performance, rather than considering only a single set of rewards.
+    This introduces some \emph{hysteris} into the system.
+\end{enumerate}
+
+Our aim with the pool ranking algorithm is to address all these concerns.
+
+\begin{comment}
+  Is this a new proposal, or just a description of the current status quo?
+\end{comment}
 
 \section{Hit Rate}
 
-The pool operator is the only one who knows their
-hit rate since others do not have access to their VRF calculation.
-Others can only try to divine it using the public knowledge of the
-pool's relative stake and number of blocks produced.
+As described above, the \emph{hit rate} for some time period $T$ is the ratio of expected to actual blocks
+that the pool produces during that time period.
+
+$$
+\begin{array}{l}
+  \begin{array}{lclp{3in}}
+    Hit~Rate_T &=& B_{Actual,T} / B_{Expected,T} \\
+  ~~ \textit{where} \\
+  \end{array}
+\\
+~~~~
+  \begin{array}{lcp{3in}}
+      B_{Actual,T} &=& \text{The actual number of blocks that a pool}\\
+      && \text{has produced in time T}\\
+      B_{Expected,T} &=& \text{The actual number of blocks that a pool}\\
+      && \text{was expected to produce in time T}
+  \end{array}
+\end{array}
+$$
+
+Note that the pool operator is the only one who knows the \emph{actual} pool
+\emph{hit rate}, since only they have access to their internal VRF calculation.
+Other actors can only estimate it using public knowledge of the
+pool's relative stake and the number of blocks that it has produced.
 
-\subsection{Estimation}
+\subsection{Hit Rate Estimation}
 
 Each slot, a stake pool with relative (active) stake $\sigma_a$ has probability
 \[ 1 - (1-f)^{\sigma_a} \]
 of producing a block, where $f$ is the active slot coefficient.
 We can use this to estimate the hit rate.
-
-We represent such an estimate as a probability distribution on the interval $[0,1]$.
+%
+We represent such an estimate as a probability distribution on the interval $[0,1]$.\todo{Are you assuming a Gaussian distribution?}
```

Comment:
it's binomial

Reply:
It's none of these!
It's a product of rescaled beta distributions of different sizes. I don't know of a "nice" way to index this family of distributions, which is why we're representing them using samples.